templates/taxonomy-topic.php

```diff
@@ -87,33 +88,10 @@
 				<?php
 				// Generate Related FAQs markup
 				$related_posts = UCF_FAQ_Common::get_related_faqs( $tags, $faqs );
+				echo UCF_FAQ_Common::display_related_faqs( $related_posts, get_field( 'related-faq-title', $topic ), $faq_atts );
 
-				foreach ( $related_posts as $post ) {
-					$related_faq_html .= UCF_FAQ_Common::display_faq( $post, $faq_atts );
-				}
-				?>
-
-				<?php
-				// Display Related FAQs
-				if ( $related_faq_html ) :
-				?>
-					<?php if ( $related_faq_title ): ?>
-					<h2 class="<?php UCF_FAQ_Config::add_athena_attr( 'h4 pt-4' ); ?>">
-						<?php echo wptexturize( $related_faq_title ); ?>
-					</h2>
-					<?php endif; ?>
-
-					<?php echo $related_faq_html; ?>
-				<?php endif; ?>
-
-				<?php
-				// Display CTA Footer
-				if ( $cta_text && $cta_url ):
+				echo UCF_FAQ_Common::display_footer_cta( $cta_text, $cta_url );
```

Comment:
I think a comment got lost here; could you re-add `// Display CTA Footer` above line 93?